components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```diff
@@ -4867,6 +4867,18 @@ definitions:
         description: |
           Number of APIs returned.
         example: 1
+      next:
```

Comment:
Pagination object already contains next and previous values. Shall we reuse it without newly adding next and previous?

Reply:
Fixed